config.example.js

```diff
@@ -1,7 +1,8 @@
 var port = process.env.PORT || 3000;
 
-// Support text emojis and almost all unicode emojis / symbols
-var emoji = "[\u2190-\u2BFF]|\ud83c[\udf00-\udfff]|\ud83d([\udc00-\ude4f]|[\ude80-\udeff])";
+// Support text emojis and all unicode emojis / symbols (as of Dec 2018)
+// See: https://www.regextester.com/106421
```

Comment:
Wow. That is awesome! You should share that!
![image](https://user-images.githubusercontent.com/6876047/84962185-17a0ab80-b0bb-11ea-9806-42baf3c1ef40.png)

Reply:
I didn't make it, any idea who to attribute here @ianrohde?

Reply:
["Dan's Tools"](https://www.regextester.com/106421) lol